index.html

```diff
@@ -199,53 +199,27 @@ <h1 class="foot-title">
           I'm always interested in hearing about new projects, so if you'd like
           to chat please get in touch.
         </h1>
-        <form
-          action="https://formspree.io/f/xbjewgnl"
-          method="post"
-          class="contact-form-input"
-        >
-          <ul class="form-input">
-            <li class="input-name">
-              <input
-                type="text"
-                name="name"
-                id="name"
-                maxlength="30"
-                placeholder="Full name"
-                required
-              />
-            </li>
-            <li class="input-full-name">
-                        <input type="text" class="full-name-bar" name="first_name" id="first_name" maxlength="30" placeholder="First name"
-                            required>
-                        <input type="text" class="full-name-bar" name="last_name" id="last_name" maxlength="30" placeholder="Last name"
-                            required>
-                    </li>
-            <li class="input-email">
-              <input
-                type="email"
-                name="email"
-                id="email"
-                placeholder="Email address"
-                required
-              />
-            </li>
-            <li class="input-text">
-              <textarea
-                name="text_area"
-                id="text_area"
-                cols="30"
-                rows="10"
-                maxlength="500"
-                placeholder="Hey I’d like to connect to..."
-                required
-              ></textarea>
-            </li>
-            <li class="submit-button">
-              <button type="submit">Get in touch</button>
-            </li>
-          </ul>
-        </form>
+       <form action="https://formspree.io/f/xbjewgnl" method="post" class="contact-form-input" id="form_id" onsubmit="validateForm(event)">
+  <ul class="form-input">
+    <li class="input-name">
+      <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required />
+    </li>
+    <li class="input-full-name">
+      <input type="text" class="full-name-bar" name="first_name" id="first_name" maxlength="30" placeholder="First name" required>
+      <input type="text" class="full-name-bar" name="last_name" id="last_name" maxlength="30" placeholder="Last name" required>
+    </li>
+    <li class="input-email">
+      <input type="email" name="email" id="email" placeholder="Email address" required />
+    </li>
+    <li class="input-text">
+      <textarea name="text_area" id="text_area" cols="30" rows="10" maxlength="500" placeholder="Hey I’d like to connect to..." required></textarea>
+    </li>
+    <li class="submit-button">
+      <button type="submit">Get in touch</button>
+      <span id="error-message" class="error-message"></span>
```

Comment:
- [ ] When trying to submit the form it doesn't work, please have a look at this because it seems that the form is not correctly setup. it would be good to start by making sure the contact section is not in the footer but in it's own section. after that please make sure to use the correct indentation this will make it a lot easier to find any discrepancies. when your HTML file is setup correctly then you can make sure that first the form submission via formspree is setup correctly and after that is working implement the JS to check for capitalization/lowercase.